src/pages/Dashboard/Modals/SupplyWithdraw/index.tsx

```diff
@@ -255,6 +255,7 @@ const SupplyWithdrawModal: React.FC<ISupplyWithdrawUiProps> = props => {
   // @TODO: elevate state so it can be shared with borrow and supply markets
   const isXvsEnabled = true;
   const borrowBalanceCents = userTotalBorrowBalance.multipliedBy(100);
+  // @TODO: include all assets in calculation of yearly earnings
```

Comment:
Why would we want to see the yearly earnings for all assets when we are supplying a single asset? It seems like you would want to see yearly earnings for the specific asset you are supplying to inform your supply decision

Reply:
That's how I understood the designs, I'll check with everyone.